CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.3.1
+
+### Bug fixes
+
+* Fixed NPE problem happened in SharedRealm.finalize() (#3730).
```

Comment:
doh .... too late ... it is all your fault, you comment 10 seconds earlier !! I will fix it next time when modify the changelog.